standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/utils/JavaUtils.java

```diff
@@ -34,4 +43,41 @@ public static ClassLoader getClassLoader() {
     }
     return classLoader;
   }
+
+  @SuppressWarnings(value = "unchecked")
+  public static <T> Class<? extends T> getClass(String className, Class<T> clazz)
+      throws MetaException {
+    try {
+      return (Class<? extends T>) Class.forName(className, true, getClassLoader());
+    } catch (ClassNotFoundException e) {
+      throw new MetaException(className + " class not found");
+    }
+  }
+
+  /**
```

Comment:
these 3 methods seem to be copy&paste from original JavaUtils.  how will this be kept in sync?

Reply:
Since standalone-metastore can't depend on things in common I'm forced to copy certain things.  I agree it isn't optimal.

Reply:
Can Hive then use the ones from the new module?

Reply:
Maybe, but keep in mind that this may split out into its own project someday.  Given that I've tried to make the rest of Hive only depend on things that were metastore specific.  Things like class instantiation seemed to make more sense for each project to have.